pages/blog/index.vue

```diff
@@ -43,11 +43,20 @@
             {{ blogCategory }}
           </h2>
         </div>
-        <div>
+
+
+        <div class="search-input-container">
+          <span class="search-icon ">
+            <img
+              src="~/assets/icons/iconSearch.svg"
+              alt="Search Icon"
+            >
```

Comment:
Let's make this clickable so that a user can click on the search icon to perform the search function.